### ISSUE TITLE
Cannot read property 'generate' of undefined

### ISSUE THREAD

**Issue by sandromartis** [json-schema-faker/json-schema-faker#599]
I sometimes get the problem that `jsf` is not defined. The strange thing is that it is quite random. Not sure if it has to do with `jsf` or with my setup.

I have the following code and I use it in Jest tests:
```
import jsf from 'json-schema-faker';

const generatedData = jsf.generate(someSchema);
```

I'm calling `jsf.generate` many many times in my test scripts. The weird thing is that I cannot reproduce the failure.
The error `Cannot read property 'generate' of undefined` shows up pretty randomly.
Most of the `jsf.generate` calls work fine, but then randomly I get the error message.

It does not matter what kind of schema I use.

Any idea why this is happening?

**Comment by pateketrueke** [comment]
I think we need to understand how you run your tests, are you using esm or something to make `import` work?

**Comment by sandromartis** [comment]
Thanks for the fast reply. I'm using Jest with the following `jest.config.js`:
```
module.exports = {
  preset: 'ts-jest',
};

```

And this `tsconfig.json`:
```
{
  "compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "moduleResolution": "node",
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "outDir": "dist/app",
    "strict": true,
    "strictNullChecks": true,
    "allowJs": true
  },
}
```

Unfortunately, I'm not able to produce a minimal working example. The error is hard to reproduce. Almost all of the `jsf.generate` calls work fine, but then one or two fail. I have multiple tests that call the same functions which are calling `jsf.generate`. Most of the time they run through fine, but sometimes one or two of the `jsf.generate` calls fail with the above error message.
I'll let you know once I have more information.

**Comment by sandromartis** [comment]
Hey @pateketrueke 

I did some more digging and realized that the error is actually coming from `sub.generate`:
https://github.com/json-schema-faker/json-schema-faker/blob/48bd5673d563ecb4427d7083b55d52ef685ad72c/src/lib/core/run.js#L131

It seems that the issue is already known and a fix has been merged: PR #594 

Could you publish to npm?

**Comment by pateketrueke** [comment]
Yes, so much thanks for your feedback!
